README.md

```diff
@@ -79,3 +80,8 @@ applications are running the functional tests can be run with
 `scripts/run_tests.sh local`
 
 (or you can substitute `local` with the name of whatever `scripts/envs/*.sh` environment file you're using.
+
+### Let the tests email Mandrill
+
+To stub out calls to the Mandrill email API, ensure you set the `DM_MANDRILL_API_KEY` environment variable
+to the key for preview when running the admin app.
```

Comment:
I think "a test key" would be better than "the key for preview". Sometimes preview might have emails turned on (e.g. I plan to do that soon for testing the new buyer account creation flow).